Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -111,28 +111,24 @@ void App::Connect(int connectionId, Platform::Object^ target) {
 }
 
 void App::OnLaunched(LaunchActivatedEventArgs^ args) {
-    if (EbrApplicationLaunched(args) == true) {
-        _ApplicationMainLaunch(ActivationTypeNone, nullptr);
-    }
+    EbrApplicationActivated(args);
```

Comment:
Suspend/Resume events are being called properly

---
In reply to: [72308899](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72308899) [](ancestors = 72308899,72308770)